docs/source/alfacase_definitions/CaseDescription.txt

```diff
@@ -6,6 +6,8 @@
 
         class CaseDescription
             name: \ :class:`Optional <typing.Optional>`\[str] = None
+            alfasim_version_info: \ :class:`Optional <typing.Optional>`\[AlfasimVersionInfo] = None
```

Comment:
Maybe try
```suggestion
            alfasim_version_info: \ :class:`Optional <typing.Optional>`\[\ :class:`AlfasimVersionInfo <AlfasimVersionInfo>`\] = None
```

Reply:
Problem is that this file is generated. If I manually edit it, the build will fail.

Reply:
> Problem is that this file is generated. If I manually edit it, the build will fail.

Given this is generated to proper way would be fixing the generator.

Reply:
> Given this is generated to proper way would be fixing the generator.

Of course.

I tried what you suggested. You were right as to what was missing, and this is what it ended up like:
```python
elif is_attrs(ref_value):
    name = attrs_formatted(ref_value)
```